widgetsnbextension/src/widget_output.js

```diff
@@ -66,7 +66,7 @@ var OutputView = widgets.DOMWidgetView.extend({
     },
 
     render: function(){
-        let renderOutput = (outputArea) => {
+        var renderOutput = function(outputArea) {
             this.output_area = new outputArea.OutputArea({
```

Comment:
We'll need to capture this, like `var that = this`, and then use `that` inside the function instead of `this`.